.github/workflows/firebase-rules.yml

```diff
@@ -0,0 +1,69 @@
+# SPDX-FileCopyrightText: Copyright 2025 Opal Health Informatics Group at the Research Institute of the McGill University Health Centre <john.kildea@mcgill.ca>
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# This workflow is explained in `docs/source/ci-cd-workflows.md`; please keep that documentation file up to date when making changes here.
+
+name: Firebase Rules
+# Default to dev when running automatically (see also "env" below)
+run-name: Deploying Firebase rules for ${{ inputs.ENVIRONMENT || 'dev' }} 🚀
```

Comment:
This could be the name for the `deploy` job below which can then use `${{ env.ENVIRONMENT }}`

Reply:
The reason I added it here was to have a descriptive title on the GitHub Actions page. I think it's helpful to be able to see at a glance the environment linked to each workflow. Here's an example from my tests:

![github-actions-example](https://github.com/user-attachments/assets/1c871ff5-18d1-41aa-a52d-edcf4779bc96)

Reply:
Nice!